src/Magnum/ImGuiIntegration/Integration.h

```diff
@@ -57,9 +57,25 @@ Example usage:
 #include "Magnum/ImGuiIntegration/visibility.h" /* defines IMGUI_API */
 
 #include <imgui.h>
+#include <Corrade/Containers/StringView.h>
 #include <Magnum/Types.h>
 #include <Magnum/Math/Vector.h>
 
+#ifndef DOXYGEN_GENERATING_OUTPUT
+#ifdef IMGUI_HAS_IMSTR
+namespace Corrade { namespace Containers { namespace Implementation {
+
+template <> struct StringViewConverter<const char, ImStrv> {
+    static StringView from(const ImStrv& other) {
+        return StringView{other.Begin, other.length()};
+    }
+    static ImStrv to(StringView other) { return ImStrv{other.begin(), other.end()}; }
+};
+
+}}}
+#endif
+#endif
```

Comment:
Instead of closing `#ifndef DOXYGEN_GENERATING_OUTPUT` here, you could just move the below `#ifndef` above, including the comment.